.pack.mk

```diff
@@ -27,10 +27,28 @@ prepare:
 
 package: prepare
 	if [ "${TARANTOOL_SERIES}" != "1.10" ]; then \
-		if [ -n "${GIT_TAG}" ]; then \
-			export VERSION="$$(echo ${GIT_TAG} | sed 's/-/~/')"; \
+		if [ "${OS}" = "alpine" ]; then \
+			if [ -n "${GIT_TAG}" ]; then \
+				if $$(echo ${GIT_TAG} | grep -q "entrypoint"); then \
+					export VERSION="$$(echo ${GIT_TAG} | sed 's/-entrypoint//')_alpha0_p0"; \
```

Comment:
Not sure about `_p0`. 
Also, it can be just `sed 's/-entrypoint/_alpha0/`.

Reply:
OK